debian/install

```diff
@@ -1,3 +1,4 @@
 ubuntu-advantage usr/bin/
 keyrings/*.gpg usr/share/keyrings/
+modules/* usr/share/ubuntu-advantage/modules
```

Comment:
Hm, I think the directory should be named after the package, i.e., ubuntu-advantage-tools, not just ubuntu-advantage.

Reply:
I kinda disagree, the directory contains modules for the script, there is not necessarily a 1:1 mapping between debian packages and directories under `/usr/share` (see `zfsutils-linux: /usr/share/zfs` for instance).
There could be other scripts in the same package that could have each their own subdirectory under `/usr/share`

Reply:
I pinged in #ubuntu-devel, let's see if there is a reply by tomorrow

Reply:
> there is not necessarily a 1:1 mapping between debian packages and directories under `/usr/share`

Agreed.

> There could be other scripts in the same package that could have each their own subdirectory under `/usr/share`

I don't think we should go down that route in this package. That just pollutes `/usr/share` for the sake of it. More commonly, when the packager is in control (as is the case here with a native package), we try to use the hierarchy more to avoid potential future conflicts which can be messy to fix later. It'd be cleaner for us to arrange everything this entire project needs to do into a single directory under `/usr/share`, using subdirectories of _that_ directory if needed.

I don't think it'd be a problem to use `/usr/share/ubuntu-advantage` for all of everything to do with "Ubuntu Advantage", including this package but also any other related package, because I don't see a potential conflict with any other package and I see it as a reasonable and responsible use of the global namespace. But by that logic multiple directories under `/usr/share/` would then be inappropriate, and I'd be unhappy if you used this as a justification now and then somebody added a `/usr/share/ubuntu-advantage-something-else` also later.

Really this is an Archive Admin question though. This is just my opinion and shouldn't be treated as any kind of approval, disapproval or decision.

Reply:
@basak thanks for the extensive argument. I'd then suggest to go with @panlinux's initial suggestion and use `/usr/share/ubuntu-advantage-tools`. If we happen to add more scripts with their own modules, we can move these and end up with something  `/usr/share/ubuntu-advantage-tools/<script>/modules` for each script. I don't think it's doing it now.